shared/src/native-lib/coreclr/src/README.md

```diff
@@ -1,14 +1,7 @@
-The files here were copied from https://github.com/dotnet/runtime/tree/v5.0.5/src/coreclr/src.
+The files here were copied from https://github.com/dotnet/runtime/tree/v7.0.0/src/coreclr/src.
```

Comment:
Can you update the README with the manual changes in the coreclr source please ? ;)

Reply:
✅ done